ChromiumHtmlToPdfLib/Browser.cs

```diff
@@ -759,24 +759,15 @@ public void Dispose()
 
         CloseAsync(CancellationToken.None).GetAwaiter().GetResult();
 
-        if (_pageConnection != null)
-        {
-            _pageConnection.Dispose();
-            _pageConnection = null;
-        }
-
-        if (_browserConnection != null)
-        {
-            _browserConnection.Dispose();
-            _browserConnection = null;
-        }
+        _pageConnection.Dispose();
+        _browserConnection.Dispose();
```

Comment:
removed nullability from fields as they don't add any value and mess nullability analysis (here and in couple of places below)